src/index.js

```diff
@@ -124,6 +125,12 @@ async function startApp() {
           enableSentry,
           isProduction,
         }),
+        formatResponse: resp => {
+          if (req.query.crunch && resp.data && !resp.data.__schema) {
```

Comment:
Only apply when

1. `crunch` is supplied as a query param
2. There's actual data to return
3. The current response isn't an introspection query